core/chains/evm/config/config_test.go

```diff
@@ -63,7 +63,7 @@ func TestChainScopedConfig(t *testing.T) {
 		gcfg2 := configtest.NewGeneralConfig(t, func(c *chainlink.Config, s *chainlink.Secrets) {
 			overrides(c, s)
 			c.EVM[0].KeySpecific = v2.KeySpecificConfig{
-				{Key: ptr(ethkey.EIP55AddressFromAddress(randomOtherAddr)),
+				{Key: testutils.Ptr(ethkey.EIP55AddressFromAddress(randomOtherAddr)),
```

Comment:
So I actually avoided this pretty deliberately during implementation since it seems way less readable and seems like it's doing more than it actually is. Plus, the go proverb:
> [A little copying is better than a little dependency.](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s)

What about keeping a local variable so the call sites are still just `ptr(`?
```go
var ptr = testutils.Ptr
```

Reply:
sure, we can do that. 

i dislike having so many definitions, but i agree that the inline is easier to read as `ptr`.

Reply:
unfortunately, this variable assignment doesn't work with generics

`var ptr = testutils.Ptr` => https://pkg.go.dev/golang.org/x/tools/internal/typesinternal#WrongTypeArgCount

and providing a type to Ptr defeats the generic.

`var ptr = testutils.Ptr[any]` => https://pkg.go.dev/golang.org/x/tools/internal/typesinternal#IncompatibleAssign

we can abandon this for now, though i would like to reduce the duplicate code